### PR TITLE
Network manager

### PR DESCRIPTION
**Pull request by chrisjws**

### CODE REVIEW

File: Community/network_manager/network_manager_page.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+def network_manager_network_manager_page_form():
+    form = GenericFormTemplate()
+    if form.validate_on_submit():
+        user = g.user._username
+        bv = g.user.get_api().get_by_object_types(user, ['IP4Block'])
+        for b in bv:
+            block = b
+            break
+        else:
+            flash("This doesn't work, you has no block.")
```
Comment (steven0784) [inline Community/network_manager/network_manager_page.py]: This doesn't work, you have* no block.

Comment (chrisjws) [reply Community/network_manager/network_manager_page.py]: I can haz cheeseburger, though.

File: Community/network_manager/network_manager_page.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        user = g.user._username
+        bv = g.user.get_api().get_by_object_types(user, ['IP4Block'])
+        for b in bv:
+            block = b
+            break
+        else:
+            flash("This doesn't work, you have no block.")
+            app.logger.error("This doesn't work, you have no block.")
+        n = block.get_next_available_ip_range(form.network_size.data, "IP4Network")
+        n.name = "%s | %s" % (form.network_location.data, form.network_name.data)
```
Comment (xiax) [inline Community/network_manager/network_manager_page.py]: This won't actually set the name of the network, you want to use something like n.set_name("%s | %s" % (form.network_location.data, form.network_name.data))

Comment (xiax) [reply Community/network_manager/network_manager_page.py]: Nevermind, this does work.

**Pull request merged**